lib/node/RequireChunkLoadingRuntimeModule.js

```diff
@@ -35,6 +35,23 @@ class RequireChunkLoadingRuntimeModule extends RuntimeModule {
 		const hasJsMatcher = compileBooleanMatcher(
 			chunkGraph.getChunkConditionMap(chunk, chunkHasJs)
 		);
+
+		let rootOutputDir = "'./'";
+		if (chunk.hasEntryModule()) {
+			const outputName = this.compilation.getPath(
+				this.compilation.outputOptions.filename + "",
+				{
+					chunk,
+					contentHashType: Object.keys(chunk.contentHash)[0]
```

Comment:
Adding `contentHasType` fixes calls to `getPath()` that were failing with:
```
    chunkName = main; chunkEntry = true; chunkInitial = true; file = bundle10.f35888f8cebc60445257.js; message = Path variable [contenthash] not implemented in this context: bundle10.[contenthash].js; chunkId = 179; details = Error: Path variable [contenthash] not implemented in this context: bundle10.[contenthash].js
```

Not sure if it's the proper fix / or place. Can you assist?

Reply:
contentHashType should be `"javascript"`